pype/plugins/global/load/delete_old_versions.py

```diff
@@ -0,0 +1,430 @@
+import traceback
+import collections
+import os
+import uuid
+
+import clique
+from pymongo import UpdateOne
+import ftrack_api
+
+from avalon import api, style
+from avalon.vendor.Qt import QtWidgets, QtCore
+from avalon.vendor import qargparse
+from avalon.api import AvalonMongoDB
+import avalon.pipeline
+from pype.api import Anatomy
+
+
+class DeleteOldVersions(api.Loader):
+
+    representations = ["*"]
+    families = ["*"]
+
+    label = "Delete Old Versions"
+    icon = "trash"
+    color = "#d8d8d8"
+
+    options = [
+        qargparse.Integer(
+            "versions", default=2, min=1, help="Versions to keep:"
+        ),
+        qargparse.Boolean("publish", help="Remove publish folder:")
```

Comment:
To keep this clear to the user I'd change the attribute names to nicer `versions to keep` and `delete folder`. I know the behaviour and caught myself mistaking whether it's version to keep or delete multiple time during testing.